doc/data-structures.rst

```diff
@@ -48,7 +48,7 @@ The :py:class:`~xarray.DataArray` constructor takes:
   :py:class:`~pandas.Series`, :py:class:`~pandas.DataFrame` or :py:class:`~pandas.Panel`)
 - ``coords``: a list or dictionary of coordinates
 - ``dims``: a list of dimension names. If omitted, dimension names are
-  taken from ``coords`` if possible.
+  taken from ``coords`` if possible (deprecated).
```

Comment:
Rather than "if possible" I would say "if coords was provided as a list"

(I think that's the only case where it's supported now.)